operator/src/main/java/oracle/kubernetes/operator/helpers/CallBuilder.java

```diff
@@ -594,6 +600,40 @@ public VersionInfo readVersionCode() throws ApiException {
     return DISPATCHER.execute(factory, requestParams, helper);
   }
 
+  /**
+   * Execute a synchronous call with a retry on failure.
+   * @param call The call
+   * @param retryDelaySeconds Retry delay in seconds
+   * @param <T> Call return type
+   * @return Results of operation, if successful
+   * @throws Exception Exception types other than ApiException, which will cause failure
+   */
+  public <T> T executeSynchronousCallWithRetry(Callable<T> call, int retryDelaySeconds) throws Exception {
+    T result = null;
+    boolean complete = false;
+    do {
+      try {
+        result = call.call();
+        complete = true;
+      } catch (RuntimeException re) {
+        Throwable cause = re.getCause();
```

Comment:
Does the ApiException ever get nested further down?  Is there ever a case where you need to loop through the causes looking for ApiException?